victims/models.py

```diff
@@ -48,3 +61,15 @@ def tags_and_colors(self):
 
     def is_valid(self):
         return bool(self.when)
+
+    def to_json(self):
+        items = {
+            "id": self.id,
+            "aggressor_side": self.aggressor_side,
+            "when": str(self.when),
+            "state": self.state,
+            "city": self.city,
+            "tags": self.tags,
+            "stories": [story.to_JSON() for story in self.stories],
+        }
```

Comment:
Any special reason to not use `dict(self)` and keep the descriptive verbose approach? IMHO this is technical debt.